tlatools/org.lamport.tlatools/src/tla2sany/StandardModules/Naturals.tla

```diff
@@ -12,6 +12,7 @@
 (* is defined in the static block at the beginning of the Java class.      *)
 (***************************************************************************)
 Nat       == { }
+ℕ         == Nat
```

Comment:
Naturals is the most fundamental module.  I suggest to hold off on this and gain more confidence with unicode in TLA+ files before the standard modules migrate.

Reply:
Does TLAPS handle unicode symbols in modules?

Reply:
I'll check but probably not. Perhaps it would be better to hide/embed this definition in the Java code version of the module somehow.

Reply:
How can user use these defs if they are hidden in Java code?  Also, wouldn't that cause problems with e.g. your tree-sitter parser, because it won't see the symbol?

Reply:
The tree-sitter parser only resolves symbols within a single file, it doesn't do imports (I would have to integrate with [stack graphs](https://github.com/github/stack-graphs) for that) so there wouldn't be a problem there. How I imagine it working for unicode in general is the user has a plugin for their editor similar to [this one](https://github.com/tlaplus-community/tlaplus-nvim-plugin) I made for neovim, and as they type it auto-replaces symbols with their unicode equivalents. So if they type `Nat` it will automatically replace it with `ℕ`, and then the java overload of the `Naturals` module will automatically resolve that to `Nat`.

Reply:
After poking around a bit I don't think there's a way to easily add an extra definition to the standard modules without it being in the .tla file. Unless you have any ideas?

Reply:
SANY/TLC support overriding any module, include the standard modules (somebody has to check how that works with the Java module overrides).  For the time being, adopters of the Unicode feature could perhaps provide a user-defined version of Natuals, ...?

Reply:
That would be acceptable, they can define `ℕ == Nat` at the top of their .tla file. When the change is eventually integrated into the modules themselves it would be a breaking change for them but the fix is simple enough that that's probably all right.